.github/workflows/esmeta-test262.yml

```diff
@@ -14,6 +14,8 @@ jobs:
         with:
           distribution: temurin
           java-version: 17
+      - name: Setup SBT
+        uses: sbt/setup-sbt@v1
```

Comment:
can we skip this step when sbt is already present?

Reply:
I'm not so familiar with GitHub actions that I know how to do that off the top of my head. I can figure it out but I have some other stuff to do first.

Reply:
`if: ` and then the condition; actually checking the condition might be tricky tho. doesn't have to be a blocker.

Reply:
Found something that seems like it should work.